solver/llbsolver/file/backend.go

```diff
@@ -380,19 +380,6 @@ func (fb *Backend) readUserWrapper(owner *pb.ChownOpt, user, group fileoptypes.M
 }
 
 func cleanPath(s string) (string, error) {
-	s, err := system.CheckSystemDriveAndRemoveDriveLetter(s, runtime.GOOS)
```

Comment:
refactored this since the new `cleanPath` called in `CheckSystemDriveAndRemoveDriveLetter` handles this.

Reply:
How about `cleanAndNormalizePath`?